src/cata_utility.h

```diff
@@ -503,6 +504,20 @@ bool equal_ignoring_elements( const Set &set, const Set &set2, const Set &ignore
                                           set2.upper_bound( *prev ), set2.end() ) );
 }
 
+/**
+ * Return a copy of a std::map with some keys removed.
+ */
+template<typename K, typename V>
+static std::map<K, V> map_without_keys( const std::map<K, V> &original,
```

Comment:
I am fairly sure this shouldn't be/doesn't need to be static.

Reply:
Updated, thanks!